tests/navier_stokes-chorin-temam_00.cc

```diff
@@ -0,0 +1,43 @@
+#include "pidomus.h"
+#include "interfaces/navier_stokes-chorin-temam.h"
+#include "tests.h"
+
+/**
+ * Test:     Navier Stokes interface.
+ * Method:   Direct
+ * Problem:  Non time depending Navier Stokes Equations
+ * Exact solution:
+ * \f[
+ *    u=\big( 2*(x^2)*y, -2*x*(y^2) \big)
+ *    \textrm{ and }p=xy;
+ * \f]
+ */
+
+using namespace dealii;
+int main (int argc, char *argv[])
+{
+
+  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv,
+                                                      numbers::invalid_unsigned_int);
+
+  initlog();
+  deallog.depth_file(1);
+
+  const int dim = 2;
+
+  NavierStokes<dim, dim, LADealII> energy;
```

Comment:
invece in parallelo cosa chiama? Vedendo il file di parametri, di default viene usato imex con kinsol